api/envoy/config/common/mutation_rules/v3/mutation_rules.proto

```diff
@@ -87,6 +88,12 @@ message HeaderMutationRules {
   google.protobuf.BoolValue disallow_is_error = 7;
 }
 
+// The HeaderForwardingRules structure specifies what headers are
+// allowed to be forwarded to the ext_proc service.
+message HeaderForwardingRules {
```

Comment:
Looks good now. Technically this isn't a mutation rule, it's a forwarding rule. I'd just fold this whole message back into ext_proc.proto and call it a day.

Reply:
good idea, done

Reply:
I agree that mutation_rules.proto is not right file. But, IMHO, the existence of this field in ext_authz is an exact example of such forwarding rule can be used in service callout in general.

However, I won't insist on this as the others are fine with moving into ext_proc.proto